datacompy/fugue.py

```diff
@@ -406,7 +406,7 @@ def report(
     def shape0(col: str) -> int:
         return sum(x[col][0] for x in res)
 
-    def shape1(col: str) -> int:
+    def shape1(col: str) -> Any:
```

Comment:
Why is this `Any`?

Reply:
The `res` variable from the the call to `_distributed_compare` is typed as `list[Any]`, then `first` is typed as Any et the shape1 cannot be typed more specifically than Any I believe.

Reply:
Yes, I think you're correct. It's `Any` even though used only in a way that would return `int`.

Reply:
In cases where we are certain that the output returns an `int`, we can explicitly cast it to `int` to ensure its type ?

```python
def shape1(col: str) -> int:
    return cast(int, first[col][1])
```

Reply:
ok I leave it as it is currently.